### PR TITLE
add logo for Opengl renderer.

### PR DESCRIPTION
**Pull request by ssakash**

### CODE REVIEW

**Comment by ssakash** [comment]
it looks fine. http://daman6009.home.comcast.net/~daman6009/gsdx.png

File: plugins/GSdx/GPUSettingsDlg.cpp [hunk 1]
```diff
@@ -140,12 +140,12 @@ void GPUSettingsDlg::UpdateControls()
 		bool dx9 = i == 0;
 		bool dx11 = i == 1;
 		bool sw = i >= 0 && i <= 2;
-
-		ShowWindow(GetDlgItem(m_hWnd, IDC_LOGO9), dx9 ? SW_SHOW : SW_HIDE);
```
Comment (gregory38) [inline plugins/GSdx/GPUSettingsDlg.cpp]: I think you srew up the space. Empty line must be empty. And code must be aligned.

File: plugins/GSdx/GSSettingsDlg.cpp [hunk 2]
```diff
@@ -413,6 +413,8 @@ void GSSettingsDlg::UpdateControls()
 
 		ShowWindow(GetDlgItem(m_hWnd, IDC_LOGO9), dx9 ? SW_SHOW : SW_HIDE);
 		ShowWindow(GetDlgItem(m_hWnd, IDC_LOGO11), dx11 ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_LOGOGL), ogl ? SW_SHOW : SW_HIDE);
+
```
Comment (gregory38) [inline plugins/GSdx/GSSettingsDlg.cpp]: Again use real empty line.

**Comment by gregory38** [comment]
git diff master...HEAD --check
plugins/GSdx/GPUSettingsDlg.cpp:143: trailing whitespace.
+               
plugins/GSdx/GSSettingsDlg.cpp:417: trailing whitespace.
+               
zsh: exit 2     git diff master...HEAD --check

**Comment by karasuhebi** [comment]
It still says 'D3D Internal Resolution'. Are we going to take out the 'D3D' part or what?

**Comment by ssakash** [comment]
@gregory38

the white spaces are removed. 

@karasuhebi

why not send a PR about it ?

**Comment by karasuhebi** [comment]
Because I don't feel like tracking down which file has that text in it and then PR'ing it. XD

**Comment by ssakash** [comment]
@karasuhebi

Go to GSdx.rc line no: 261. there you go, work cut down for  you :p

**Comment by myownfriend** [comment]
I feel like the "D3D Internal Resolution" text is far more in need of being fixed than the lack of OGL logo. The D3D and OGL logos aren't really needed at all anyway.

**Comment by ssakash** [comment]
I guess @karasuhebi can make the PR for the text change. if not, I'll make a PR for it.

**Comment by gregory38** [comment]
What do you think about "off-screen internal resolution"

**Comment by ssakash** [comment]
It looks fine to me but, we still need more opinions on deciding the text.

**Comment by gregory38** [comment]
Agreed. Do a poll on the forum.

**Comment by myownfriend** [comment]
Why not just "Internal resolution"?

**Comment by bositman** [comment]
Yeah I'd say just "internal resolution" would suffice. Adding extra words will only confuse ignorant noobs (if it wasn't confusing already :P)

**Comment by myownfriend** [comment]
On the topic of things being confusing, doesn't GSDX mean Graphics Synthesizer Direct X? Shouldn't the OpenGL backend technically be called GSOGL or something similar?

**Comment by ssakash** [comment]
GSDX is the plugins name. OpenGL is one of the renderers present on the plugin, it's way better than having two plugins with different names.

By the way, I have made a poll on the forums based on the replacement of the words. I guess we could come to a conclusion after the results ?

**Comment by karasuhebi** [comment]
I say just drop the D3D and leave it 'Internal Resolution'. I'm not PR'ing it though. You can do it @ssakash

**Comment by bositman** [comment]
@myownfriend Yeah that's correct but when Gabest started it, it was supposed to be DX only. Changing that would be even more confusing than making things clearer...
I don't think we need a forum poll for that though :P

**Comment by refractionpcsx2** [comment]
I don't know why a poll was even suggested, the solution was obviously just to drop the D3D part.

**Comment by ssakash** [comment]
I guess I will go with "Internal resolution" choice then. Though, off-screen internal resolution does sound even more precise. Anyway, everyone voted for the "internal resolution" so, I guess that's the way to go.

**Comment by gregory38** [comment]
```
I don't know why a poll was even suggested, the solution was obviously just to drop the D3D part.
```
Personally I found internal unclear. People often uses off-screen in smart phone review, so why not.

Anyway I'm inclining toward the majority.

**Comment by ssakash** [comment]
The majority of the votes were for "Internal Resolution" hence, I have used that in the commit.

**Comment by mirhl** [comment]
Mhh.. what about "rendering resolution" name?
It somewhat recalls modern gpu features like oversampling.

**Comment by karasuhebi** [comment]
I know it's probably too late now but I think Rendering Resolution is a perfect fit. Thanks for the suggestion @mirhl

**Comment by ssakash** [comment]
cool, I can still change it if more people agree on a certain thing.

**Comment by mirhl** [comment]
Hey, I failed to met ssakash's deadline for compiling, this is the minimum  :p
Btw I almost set up VS2015 on my computer at home.. So @ssakash ping me at any time :)

**Comment by ssakash** [comment]
I need your help one more time, I'll contact you then. ;)

**Pull request merged**